src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -325,16 +326,12 @@ class SeekBar extends Slider {
     }
     this.player_.scrubbing(false);
 
-    /**
-     * Trigger timeupdate because we're done seeking and the time has changed.
-     * This is particularly useful for if the player is paused to time the time displays.
-     *
-     * @event Tech#timeupdate
-     * @type {EventTarget~Event}
-     */
-    this.player_.trigger({ type: 'timeupdate', target: this, manuallyTriggered: true });
```

Comment:
why remove this?

Reply:
During my tests, it looks like this is not needed anymore but maybe it is used somewhere... I didn't check Flash but not sure if Flash is still in use... However, if you think that I shouldn't remove it I can add it again. any additional thoughts?

Reply:
Yeah, I'd be wary to remove it without dropping older browser versions and what not, which we won't do except in a major version.

Reply:
Yeah, I'd be wary to remove it without dropping older browser versions and what not, which we won't do except in a major version.

Reply:
Yeah, It sounds sane. I have just re-added the trigger for timeupdate event again.

Reply:
Thanks!